### DISCUSSION TITLE
请问如何恢复默认字体

### DISCUSSION THREAD

**Discussion by lincTHU** [GpointChen/FFXIVChnTextPatch-GP#208] [General]
打了4k字体包，但是字体有点不习惯，想恢复成原来的字体
另外请问4k字体包的制作有资料可以参考吗？想自己换个字体试试

**Comment by GpointChen** [comment]
> 打了4k字体包，但是字体有点不习惯，想恢复成原来的字体 

把`resource/font`裡面的檔案刪除，在漢化時就可以不打字體包補丁，不過可能會導致大量中文字無法顯示。

> 另外请问4k字体包的制作有资料可以参考吗？想自己换个字体试试

[這個插件](https://github.com/Soreepeong/XivAlexander)有附帶製作字體的功能。

詳細流程可以參考別人寫的教學：
https://home.gamer.com.tw/artwork.php?sn=5399504

**Comment by lincTHU** [comment]
https://github.com/Soreepeong/FFXIV-FontChanger
这个可以直接生成对应的文件 跳过alexander生成的步骤；会方便一些
不过总体来讲调字体还是挺麻烦的
